ports/libvpx/portfile.cmake

```
@@ -165,7 +164,11 @@ else()
 	elseif(VCPKG_TARGET_IS_LINUX)
         set(LIBVPX_TARGET "${LIBVPX_TARGET_ARCH}-linux-gcc")
     elseif(VCPKG_TARGET_IS_OSX)
-        set(LIBVPX_TARGET "${LIBVPX_TARGET_ARCH}-darwin17-gcc") # enable latest CPU instructions for best performance and less CPU usage on MacOS
+        if(VCPKG_TARGET_ARCHITECTURE STREQUAL arm64)
```

```suggestion
        if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
```
the quotes are necessary.

---

this is also really suspect given the above setting of LIBBPX_TARGET_ARCH

---

@strega-nil-ms, thank you, I just learned about this policy, https://cmake.org/cmake/help/latest/policy/CMP0054.html#policy:CMP0054 :)

---

There are also some other places that have the same problem. Might also require quotes.